frontend/public/locales/en/common.json

```diff
@@ -0,0 +1,196 @@
+{
```

Comment:
Some of the labels, especially in the common file, could potentially be easier to read if we format them like https://www.i18next.com/translation-function/essentials#accessing-keys.
I did not test if it was applicable.